paymentsheet/src/test/java/com/stripe/android/paymentelement/embedded/form/FakeFormActivityStateHelper.kt

```
@@ -20,10 +22,22 @@ internal class FakeFormActivityStateHelper : FormActivityStateHelper {
             )
         )
 
-    override fun update(confirmationState: ConfirmationHandler.State) {
+    override fun updateConfirmationState(confirmationState: ConfirmationHandler.State) {
         updateTurbine.add(confirmationState)
     }
 
+    override fun updateMandate(mandateText: ResolvableString?) {
+        // NO-OP
```

Seems like we're missing tests then, right?

---

Can we throw an error instead of no op if they're not called?

---

Updated